xarray/core/groupby.py

```diff
@@ -191,7 +191,10 @@ def _unique_and_monotonic(group):
         return True
     else:
         index = safe_cast_to_index(group)
-        return index.is_unique and index.is_monotonic
+        if index.dtype == "O":
+            return index.is_unique
```

Comment:
Could I ask why this change? Does `is_monotonic` not work for object indexes?

Reply:
It's meaningless for object indices I think. This is how I found this:

``` python
>>> xr.tutorial.scatter_example_dataset().indexes["w"]

 Index(['one', 'two', 'three', 'five'], dtype='object', name='w')

 >>> xr.tutorial.scatter_example_dataset().indexes["w"].is_monotonic

 False
```

Reply:
But isn't that because that index isn't monotonic? e.g.

```python

In [5]:  xr.tutorial.scatter_example_dataset().indexes["w"].sort_values().is_monotonic
Out[5]: True
```

Reply:
my bad. :+1: I'll fix it tomorrow